Gibe.Navigation.Umbraco/NodeTypes/SettingsNodeType.cs

```diff
@@ -1,15 +1,45 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
+using Moq;
+using NUnit.Framework;
 using Umbraco.Core.Models;
 
 namespace Gibe.Navigation.Umbraco.NodeTypes
 {
 	public class SettingsNodeType : INodeType
 	{
+		private readonly string _docTypeAlias;
+
+		public SettingsNodeType(string docTypeAlias = "site")
+		{
+			_docTypeAlias = docTypeAlias;
+		}
+
 		public IPublishedContent FindNode(IEnumerable<IPublishedContent> rootNodes)
 		{
-			return rootNodes.First(r => r.DocumentTypeAlias == "site");
+			return rootNodes.First(r => r.DocumentTypeAlias == _docTypeAlias);
+		}
+	}
+
+	[TestFixture]
+	internal class SettingsNodeTypeTests
+	{
+		[Test]
+		public void FindNode_Returns_First_Node_Of_Type_Site()
```

Comment:
Name would seem incorrect given the changes. Is the behaviour right? Does having two top level "site" nodes make sense? If we used this in bristol sport, for example, it would seem like we'd super want it to fail loudly because it can't handle that kind of setup.